src/numpy-stubs/_core/umath.pyi

```diff
@@ -746,6 +747,94 @@ class _Call21Bool(Protocol):
         **kwds: Unpack[_Kwargs3],
     ) -> np.bool | NDArray[np.bool]: ...
 
+@type_check_only
+class _Call21Float(Protocol):
+    @overload  # (float, float) -> float64
+    def __call__(
+        self,
+        x1: float,
+        x2: float,
+        /,
+        out: None = None,
+        *,
+        dtype: _DTypeLikeFloat | None = None,
+        **kwds: Unpack[_Kwargs3],
+    ) -> np.float64: ...
+    @overload  # (scalar, scalar) -> float
+    def __call__(
+        self,
+        x1: _FloatLike_co,
+        x2: _FloatLike_co,
+        /,
+        out: None = None,
+        *,
+        dtype: _DTypeLikeFloat | None = None,
+        **kwds: Unpack[_Kwargs3],
+    ) -> np.floating: ...
+    @overload  # (array-like, array-like, out: T) -> T
+    def __call__(
+        self,
+        x1: _ArrayLikeFloat_co,
+        x2: _ArrayLikeFloat_co,
+        /,
+        out: _Out1[_ArrayT],
+        **kwds: Unpack[_Kwargs3],
+    ) -> _ArrayT: ...
+    @overload  # (array-like, array) -> Array[float64]
+    def __call__(
+        self,
+        x1: CoFloat64_nd,
+        x2: ToFloat64_1nd,
+        /,
+        out: None = None,
+        *,
+        dtype: None = None,
+        **kwds: Unpack[_Kwargs3],
+    ) -> NDArray[np.float64]: ...
+    @overload  # (array, array-like) -> Array[float64]
+    def __call__(
+        self,
+        x1: NDArray[np.float64] | _NestedSequence[float],
+        x2: _ArrayLikeFloat_co,
```

Comment:
This is the mirror image of the overload above, so it should also use the `_numtype` aliases